Comments/ViewModels/Comment.cs

```diff
@@ -25,7 +25,9 @@ public Comment(int locationId, string sourceUri, string htmlElementId, string ra
 	        if (!(comment.Status is null))
 		        Status = new Status(comment.Status);
 	        StatusId = comment.StatusId;
-        }
+			if(!(comment.OrganisationUser is null))
```

Comment:
while valid syntax, i don't think it's very readable. i'd personally use: 

if (comment.OrganisationUser != null)

or at a push, maybe:

if (comment.OrganisationUser is not null)

but feel free to leave it, as it matches the above Status check. (if you do change it though, change the Status too please).